src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -19,12 +19,16 @@
   <ItemGroup>
     <None Remove="FileProvisioning\PowerShell\profile.ps1" />
     <None Remove="FileProvisioning\PowerShell\requirements.psd1" />
+    <None Remove="runtimeassemblies-relaxed.json" />
     <None Remove="runtimeassemblies.json" />
     <None Remove="runtimes.json" />
   </ItemGroup>
   <ItemGroup>
     <EmbeddedResource Include="FileProvisioning\PowerShell\profile.ps1" />
     <EmbeddedResource Include="FileProvisioning\PowerShell\requirements.psd1" />
+    <EmbeddedResource Include="runtimeassemblies-relaxed.json">
```

Comment:
@brettsam I noticed this was removed for v3 but wasn't sure about the repercussions.. Should I go ahead and remove it?

Reply:
runtimeassemblies-relaxed **should be included** for v3. It allows us to switch to "V2 compat mode" if we need to and use this set of unification rules.

Reply:
This looks right -- where did you see it was removed?

Reply:
My bad, it's not there in v2 and is present in v3. So yes, guess this change is valid then .